<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_server_settings">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Connection settings - Server</string>
    </property>
    <widget class="QLabel" name="label_serverIP">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Addr:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_serverPort">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_serverIP">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.35</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_serverPort">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string>5000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_serverEnable">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_PID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>141</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>PID - Angle and Speed Parameters</string>
    </property>
    <widget class="QLabel" name="label_KpAngle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kp:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_KiAngle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ki:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KpAngle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KiAngle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_KdAngle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kd:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KdAngle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sendPID">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_KiSpeed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ki:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_KpSpeed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kp:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KiSpeed">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_KdSpeed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kd:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KdSpeed">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_KpSpeed">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>62</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sendPID_Zero">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Orientation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Orientation</string>
    </property>
    <widget class="QLabel" name="label_roll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Roll:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pitch">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Pitch:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_yaw">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Yaw:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_roll">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_pitch">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_yaw">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Motor">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Motor speed</string>
    </property>
    <widget class="QLabel" name="label_speedPid">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speedRun">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_speedTurn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Turn</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speedPID">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speedRun">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speedTurn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_speedLeft">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speedLeft">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_speedRight">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_speedRight">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_client_settings">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Connection settings - Client</string>
    </property>
    <widget class="QLabel" name="label_clientIP">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>IP Addr:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_clientPort">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Port:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_clientIP">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.49</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_clientPort">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5001</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clientEnable">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_restartUDP">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>Restart UDP...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
